examples/streaming/state/src/main/scala/org/apache/gearpump/streaming/examples/state/processor/CountProcessor.scala

```diff
@@ -37,7 +37,7 @@ class CountProcessor(taskContext: TaskContext, conf: UserConfig)
   }
 
   override def processMessage(state: PersistentState[Int], message: Message): Unit = {
-    state.update(message.timestamp, 1)
+    state.update(message.timestamp.toEpochMilli, 1)
```

Comment:
It's only used internally in `Subscription` and can be replaced in the future as well.